server/src/main/java/io/druid/client/CachingClusteredClient.java

```diff
@@ -146,21 +163,21 @@ public CachingClusteredClient(
   @Override
   public <T> QueryRunner<T> getQueryRunnerForIntervals(final Query<T> query, final Iterable<Interval> intervals)
   {
-    return new QueryRunner<T>()
-    {
-      @Override
-      public Sequence<T> run(final QueryPlus<T> queryPlus, final Map<String, Object> responseContext)
-      {
-        return CachingClusteredClient.this.run(queryPlus, responseContext, timeline -> timeline);
-      }
-    };
+    return (queryPlus, responseContext) -> runAndMergeWithTimelineChange(
+        query,
+        queryPlus,
+        responseContext,
+        // No change, but Function.identity() doesn't work here for some reason
+        stringServerSelectorTimelineLookup -> stringServerSelectorTimelineLookup
```

Comment:
What logical meaning does "string" have?

Reply:
just got caught in a bad refactor I think